packages/elements/src/components/metrics-table/metrics-table.component.ts

```diff
@@ -40,65 +40,95 @@ export class MutationTestReportTestMetricsTable<TFile, TMetric> extends LitEleme
     low: 60,
   };
 
+  public static styles = [tailwind];
+
+  private hasMultipleColspan = false;
+
+  public override willUpdate(changedProperties: PropertyValues) {
+    if (changedProperties.has('columns')) {
+      this.hasMultipleColspan = this.columns.some((column) => column.category === 'percentage');
+    }
+  }
+
   public render() {
     return html`${this.model
-      ? html`<table class="table table-hover table-no-top">${this.renderTableHeadRow()}${this.renderTableBody(this.model)}</table>`
-      : ''}`;
+      ? html`<div class="container overflow-auto rounded-md border border-gray-200 motion-safe:transition-max-width">
+          <table class="w-full table-auto text-left text-sm">${this.renderTableHeadRow()}${this.renderTableBody(this.model)} </table>
+        </div>`
+      : nothing}`;
   }
 
   private renderTableHeadRow() {
-    return html`<thead>
-      <th scope="col" colspan="2" style="width: 217px">
-        <div
-          ><span>File / Directory</span
-          ><a
-            href="https://stryker-mutator.io/docs/mutation-testing-elements/mutant-states-and-metrics"
-            target="_blank"
-            class="info-icon"
-            title="What does this all mean?"
-            >ℹ</a
-          ></div
-        >
-      </th>
-      ${this.columns.map((column) => this.renderTableHead(column))}
+    return html`<thead class="border-b border-gray-200 text-center text-sm">
+      <tr>
+        <th scope="col" class="py-4 px-4">
+          <div class="flex items-center justify-around">
+            <span>File / Directory</span
+            ><a
+              href="https://stryker-mutator.io/docs/mutation-testing-elements/mutant-states-and-metrics"
+              target="_blank"
+              class="info-icon float-right"
+              title="What does this all mean?"
+              >${renderEmoji('ℹ', 'info icon')}</a
+            >
+          </div>
+        </th>
+        ${repeat(
+          this.columns,
+          (column) => column.key,
+          (column) => this.renderTableHead(column)
+        )}
+      </tr>
     </thead>`;
   }
 
   private renderTableHead(column: Column<TMetric>) {
+    const classNames = this.hasMultipleColspan ? 'even:bg-gray-100' : 'odd:bg-gray-100';
```

Comment:
This does not work great for test files. I would expect the `killing` column to be light grey.
![image](https://user-images.githubusercontent.com/1828233/216955094-f0d9ce68-03fc-4d40-9b1b-0ab8c0671494.png)

Reply:
![image](https://user-images.githubusercontent.com/10114577/217206844-9c572014-1a04-4b1e-8ca8-ecc6291d90e5.png)
Fixed